tests/monotouch-test/Network/NWWebSocketOptionsTest.cs

```diff
@@ -0,0 +1,104 @@
+﻿#if !__WATCHOS__
+using System;
+using System.Collections.Generic;
+using System.Threading;
+#if XAMCORE_2_0
+using CoreFoundation;
+using Foundation;
+using Network;
+using ObjCRuntime;
+using Security;
+#else
+using MonoTouch.CoreFoundation;
+using MonoTouch.Foundation;
+using MonoTouch.Network;
+using MonoTouch.Security;
+#endif
+
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.Network {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NWWebSocketOptionsTest {
+
+		NWWebSocketOptions options;
+
+		[TestFixtureSetUp]
+		public void Init () => TestRuntime.AssertXcodeVersion (11, 0);
+
+		[SetUp]
+		public void SetUp ()
+		{
+			options = new NWWebSocketOptions (NWWebSocketVersion.Version13);
+		}
+
+		[TearDown]
+		public void TearDown ()
+		{
+			options?.Dispose ();
```

Comment:
same - remove `?`, if some test mess with it we need to know (since the order of execution is not guaranteed)